composer.json

```diff
@@ -61,9 +61,11 @@
     },
     "scripts": {
         "test:types": "phpstan analyse --ansi",
+        "test:laravel-types": "phpstan analyse --ansi --level=max --configuration ./extension.neon ./tests/Application/app",
```

Comment:
Kind of unrelated. Realized we did not run Larastan on the test application. Figured it would be a good addition to the current tests.